Engine/Engine.csproj

```diff
@@ -70,7 +70,7 @@
 
 
   <ItemGroup Condition="'$(TargetFramework)' == 'net6'">
-    <PackageReference Include="System.Management.Automation" Version="7.0.11" />
+    <PackageReference Include="System.Management.Automation" Version="7.0.13" />
```

Comment:
@andyleejordan Why not `7.2.23`? I believe 7.0.x was from days where we still had to support PowerShell 7.0. Now it's just 7.2 and after November 8, it will be just 7.4 (which will also mean we will be able to bump .NET version to 8) https://learn.microsoft.com/en-us/powershell/scripting/install/powershell-support-lifecycle?view=powershell-7.4#powershell-end-of-support-dates

Reply:
Yeah fair but we should do that for a point release, not a patch release.

Reply:
So @bergmeister I actually have to circle back and do this now 😅